scripts/batch-score.coffee

```diff
@@ -0,0 +1,50 @@
+# Description:
```

Comment:
> Our bot already has a command bot score which tells us the score of the user, i.e., the amount of appreciation received. But let's say we want to know the score of a whole batch, we'll need to manually
enter the name of each member, which is quite a tedious task. Add batchwise-score script to make this task easier. Now the user will be able to get the score of each user in a particular batch all at once by the command bot score bxx

Some comments on commit message:
1. Your commit message is wrapped around 72 characters.
2. You've added batch-score.coffee not batchwise-score.coffee. I prefer batch-score.coffee since we'll be adding some more commands and considering them `batch-score` is a better name.

> Now the user will be able to get the score of each user in a particular batch all at once by the command bot score bxx

3. This line is unnecessary since it is more of a "documentation" line. If you want you can add it to the script itself but I don't suggest so.

Reply:
I have done the necessary changes. Please check.

Reply:
Don't commit just now. Wait for @dev-ritik to review this PR.

Reply:
another nit: your oneliner starts with `Adds` change it to `Add`. We should write messages as if we are instructing the code base to do as said.

Reply:
nit: And get this through your mind. **Every line of the commit message should be less than 72 characters**